tox.ini

```diff
@@ -64,4 +65,10 @@ commands =
 usedevelop = true
 commands = pip install -U celery==4.3
            pip install -U kombu==4.5
-           py.test --cov=django_celery_results --cov-report=xml --no-cov-on-fail
+           pytest --cov=django_celery_results --cov-report=xml --no-cov-on-fail
+
+[testenv:integration]
+commands =
+    pip install -U celery==4.3
```

Comment:
try celery==4.4.0rc3

Reply:
kombu 4.6.x

Reply:
We can do that, but then, shouldn't we use the latest version for all test testenv builds, not just integration?

The easiest way to enable pre-release to be installed for all unpinned dependencies is to use the `pip_pre` tox flag: https://tox.readthedocs.io/en/latest/config.html#conf-pip_pre